paper-progress.html

```diff
@@ -27,98 +28,171 @@
 
 To change the active progress bar color:
 
-    paper-progress::shadow #activeProgress {
-      background-color: #e91e63;
+    paper-progress {
+       --paper-progress-active-color: #e91e63;
     }
 
 To change the secondary progress bar color:
 
-    paper-progress::shadow #secondaryProgress {
-      background-color: #f8bbd0;
+    paper-progress {
+      --paper-progress-secondary-color: #f8bbd0;
     }
-    
+
 To change the progress bar background color:
 
-    paper-progress::shadow #progressContainer {
-      background-color: #64ffda;
+    paper-progress {
+      --paper-progress-container-color: #64ffda;
     }
 
 @group Paper Elements
 @element paper-progress
-@extends core-range
 @homepage github.io
 -->
```

Comment:
Sorry for the late catch, but can you move the comments below the `import`s? If the comment does not appear next to the `dom-module`, it won't show up in `iron-doc-viewer` /  `iron-component-page`.